examples/chapter1/mem_example/CMakeLists.txt

```diff
@@ -29,5 +29,4 @@ target_link_libraries(${PROJ_NAME} PUBLIC ${KPSR_ROBOTICS_CORE_LIBRARIES}
 # Include directories with headers
 # ---------------------------------------------------#
 target_include_directories(
-    ${PROJ_NAME} PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/include
```

Comment:
I think we can rename this to datamux instead of mem_example? Including the directory